contrib/src/uuid.rs

```diff
@@ -0,0 +1,133 @@
+extern crate uuid as uuid_ext;
+
+use std::fmt;
+use std::str::FromStr;
+use std::ops::{Deref, DerefMut};
+use rocket::request::FromParam;
+
+/// The UUID type, which implements `FromParam`. This type allows you to accept
+/// values from the [Uuid](https://github.com/rust-lang-nursery/uuid) crate as
+/// a dynamic parameter in your request handlers.
+///
+/// # Usage
+///
+/// To use, add the `uuid` feature to the `rocket_contrib` dependencies section
+/// of your `Cargo.toml`:
+///
+/// ```toml,ignore
+/// [dependencies.rocket_contrib]
+/// version = "*"
+/// default-features = false
+/// features = ["uuid"]
+/// ```
+///
+/// The UUID type implements the Deref trait allowing you to access the
+/// underlying Uuid type.
+///
+/// ```rust,ignore
+/// #[get("/users/<id>")]
+/// fn user(id: UUID) -> String {
+///     // Use Deref to access the underlying Uuid type.
+///     expects_uuid(*id);
+///
+///     format!("We found: {}", id)
+/// }
+/// ```
+#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Debug)]
+pub struct UUID(uuid_ext::Uuid);
+
+impl UUID {
+    /// Consumes the UUID wrapper returning the underlying Uuid type.
+    ///
+    /// # Example
+    /// ```rust,ignore
```

Comment:
I did...  It ends up somehow breaking things and it suddenly stops being able to find rocket_contrib and etc. Feel free to give it a try, but I wasn't able to make it work.